### DISCUSSION TITLE
Task dependent on input existence

### DISCUSSION THREAD

**Discussion by xenobytezero** [moonrepo/moon#1453] [Q&A]
I would like to have my ESLint command at the top level `tasks.yml` , but running the task requires the presence of an `.eslintrc.cjs` file in the project.

Is it possible to have a task only run if an input is present? Otherwise I would have to have a `moon.yaml` in every project and use the tag system

**Comment by milesj** [comment]
> Is it possible to have a task only run if an input is present?

We don't support this. The inputs are only used for affected detection.

If you're trying to just check if a file exists in a project before running, you can probably wrap the eslint command in a tiny bash script.
